### DISCUSSION TITLE
Frigate Card

### DISCUSSION THREAD

**Discussion by Welsyntoffie** [blakeblackshear/frigate#13899] [Ask A Question]
I have never gotten the frigate card to work since moving to docker containers.
When I select the camera from the drop down I get an error...
`Camera initialization failed: API error whilst subscribing to events for unknown Frigate instance frigate. [Check troubleshooting](https://github.com/dermotduffy/frigate-hass-card#troubleshooting).`

I have the frigate integration installed and when selecting the camera can view is just fine in real time with status recording. I assume this is directly from frigate server. But when selecting this camera for the frigate card I get the above mentioned error.

I am running everything from docker containers.
My docker compose for frigate...

`version: "3.9"
services:
  frigate_beta14:
    container_name: frigate14
    privileged: true
    restart: always
    image: ghcr.io/blakeblackshear/frigate:0.14.1
    shm_size: "200mb"
    devices:
      - /dev/dri/renderD128:/dev/dri/renderD128
    deploy:
      resources:
        limits:
          memory: 5632M
        reservations:
          memory: 5120M
    memswap_limit: 6000M

    volumes:
      - /etc/localtime:/etc/localtime:ro
      - /home/pieter/dockerdata/frigate14/config/config.yml:/config/
      - /mnt/frigaterecording/:/media/frigate
      - type: tmpfs # Optional: 1GB of memory, reduces SSD/SD Card wear
        target: /tmp/cache
        tmpfs:
          size: 1000000000
    ports:
      - "5005:5000"
      - "5006:8971"
      - "8554:8554" # RTSP feeds
      - "8555:8555/tcp"
      - "8555:8555/udp"
      - "1984:1984"
    environment:
      - LIBVA_DRIVER_NAME=radeonsi`

![Untitled](https://github.com/user-attachments/assets/7406cf67-b625-4535-a48c-b62053b141b0)

Card config...
type: custom:frigate-card
cameras:
  - camera_entity: camera.back_yard

Am I missing more settings, tried to give frigate server URL different engines.
Tried setting the port back to 5000 in docker compose just to test but still have api error.
Just not getting it.

**Comment by hawkeye217** [comment]
This is the repo for Frigate itself, you'll likely want to file your issue over at the Frigate card repo: https://github.com/dermotduffy/frigate-hass-card

**Reply by Welsyntoffie** [reply]
Thanks for the info.
